docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -256,6 +260,37 @@ difference is that you would specify your Hibernate ORM configuration in `META-I
 </persistence>
 --
 
+== Defining entities in external projects or jars
+
+Hibernate ORM in Quarkus relies on compile-time bytecode enhancements to your entities. If you define your entities in the
+same project where you build your Quarkus application, everything will work fine. If the entities come from external projects
+or jars, you should build those jars with the Jandex Maven plugin, so that your entities will be indexed and found by Quarkus
```

Comment:
I suggest to change:

`will be indexed and found by Quarkus and enhanced`

To:

`will be indexed and enhanced by {project-name}`

Reply:
Per @emmanuelbernard we do not need {project-name} any more so can we change this to:

`will be indexed and enhanced by Quarkus`

?

Reply:
Thanks, fixed